src/autoscaler/scalingengine/server/server_test.go

```diff
@@ -144,4 +143,96 @@ var _ = Describe("Server", func() {
 
 	})
 
+	Context("when setting active schedule", func() {
+		BeforeEach(func() {
```

Comment:
You have a bunch of Before blocks but if you just focus on what really needs to change, you would just tweak what is needed per test rather than a lot of extra code.

The request should be created in the JustBeforeEach.
Expose the method.

Now you just have to tweak the urlPath, method or body.